packages/terra-paginator/docs/ProgressiveREADME.md

```diff
@@ -0,0 +1,23 @@
+# Terra Paginator
```

Comment:
Rename this headre to Terra Progressive Paginator

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/7827f90bf4179514cd22d77a2b46b608bdc8c660